src/components/common/dialogs/DeleteDialog.tsx

```diff
@@ -11,36 +11,17 @@ export interface DeleteDialogProps {
 
 export class DeleteDialog extends Component<DeleteDialogProps> {
     render() {
-        return <VisibleModal className="">
-            <div className="modal__body">
-                <DialogTitle title={this.props.title} description={this.props.description}> {this.props.children} </DialogTitle>
-                <DialogBody>{this.props.children}</DialogBody>
+        return <ConfirmationDialog>
```

Comment:
I think we can create a function which takes input like src, title, description, closeDelete, delete and return this complete component when they have same class attached

Reply:
For dialogs that requires more customisation we are using ConfirmationDialog. Have extracted out common markup in DeleteDialog.